src/components/AfterInitialMount/AfterInitialMount.tsx

```diff
@@ -1,12 +1,14 @@
-import React from 'react';
+import React, {ReactNode} from 'react';
 import {useIsAfterInitialMount} from '../../utilities/use-is-after-initial-mount';
 
 interface Props {
-  children?: React.ReactNode;
+  children?: ReactNode;
+  fallback?: ReactNode;
 }
 
-export function AfterInitialMount({children}: Props) {
+export function AfterInitialMount({children, fallback = null}: Props) {
   const isMounted = useIsAfterInitialMount();
+  const content = isMounted ? children : fallback;
 
-  return isMounted ? children : null;
+  return <React.Fragment>{content}</React.Fragment>;
```

Comment:
Typescript was having a hard time understanding the return type of the function was a react element so I wrapped it in a fragment rather than casting

Reply:
Had a play with this and while `return {content}` works in react the types are grumpy and casting doesn't seem to make it happier :(

It's annoying but i think this will have to stay for a while